frontend/express/public/javascripts/countly/vue/components/helpers.js

```diff
@@ -591,4 +591,65 @@
         }
     }));
 
+    Vue.component("cly-notification", countlyBaseComponent.extend({
+        template: '<div v-if="isModalVisible===true" :class="dynamicClasses" class="cly-vue-notification__alert-box">\n' +
+                        '<div class="bu-is-flex bu-is-justify-content-space-between">\n' +
+                            '<div class="bu-is-flex bu-is-justify-content-space-between" style="overflow-wrap:anywhere;">\n' +
+                                '<img :src="image" class="bu-mr-4 bu-my-1 bu-ml-1 alert-image">\n' +
+                                '<span class="alert-text bu-my-auto">{{text}}</span>\n' +
+                            '</div>\n' +
+                            '<div @click="closeModal" class="bu-mr-3 bu-ml-6" style="margin:auto">\n' +
+                                '<slot><i class="el-icon-close"></i></slot>\n' +
```

Comment:
So we might need to give this slot a name such as "right" :slightly_smiling_face: